packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -0,0 +1,414 @@
+        OptimismMintableERC20Factory(impl)
+            .initialize(address(0));
+
+        vm.expectRevert();
+        OptimismMintableERC20Factory(impl)
+            .initialize(address(0));
+
+        OptimismMintableERC20Factory(Predeploys.OPTIMISM_MINTABLE_ERC20_FACTORY)
+            .initialize(Predeploys.L2_STANDARD_BRIDGE);
+
+        vm.expectRevert();
+        OptimismMintableERC20Factory(Predeploys.OPTIMISM_MINTABLE_ERC20_FACTORY)
+            .initialize(Predeploys.L2_STANDARD_BRIDGE);
+
+        _verifyProxyImplementationAddress(Predeploys.OPTIMISM_MINTABLE_ERC20_FACTORY, impl);
+    }
+
+    /// @notice This contract has no initializer.
+    function _setL1BlockNumber() internal {
+        address impl = _predeployToCodeNamespace(Predeploys.L1_BLOCK_NUMBER);
+        vm.etch(
+            impl,
+            vm.getDeployedCode("L1BlockNumber.sol:L1BlockNumber")
+        );
+
+        _verifyProxyImplementationAddress(Predeploys.L1_BLOCK_NUMBER, impl);
+    }
+
+    /// @notice This contract has no initializer.
+    function _setGasPriceOracle() internal {
+        address impl = _predeployToCodeNamespace(Predeploys.GAS_PRICE_ORACLE);
+        vm.etch(
+            impl,
+            vm.getDeployedCode("GasPriceOracle.sol:GasPriceOracle")
+        );
+
+        _verifyProxyImplementationAddress(Predeploys.GAS_PRICE_ORACLE, impl);
+    }
+
+    /// @notice This contract is NOT proxied.
+    function _setGovernanceToken() internal {
+        if (!cfg.enableGovernance()) {
+            console.log("Governance not enabled, skipping setting governanace token");
+            return;
+        }
+
+        GovernanceToken token = new GovernanceToken();
+        vm.etch(Predeploys.GOVERNANCE_TOKEN, address(token).code);
+
+        bytes32 _nameSlot = hex"0000000000000000000000000000000000000000000000000000000000000003";
+        bytes32 _symbolSlot = hex"0000000000000000000000000000000000000000000000000000000000000004";
+        bytes32 _ownerSlot = hex"000000000000000000000000000000000000000000000000000000000000000a";
+
+        vm.store(
+            Predeploys.GOVERNANCE_TOKEN,
+            _nameSlot,
+            vm.load(address(token), _nameSlot)
+        );
+        vm.store(
+            Predeploys.GOVERNANCE_TOKEN,
+            _symbolSlot,
+            vm.load(address(token), _symbolSlot)
+        );
+        vm.store(
+            Predeploys.GOVERNANCE_TOKEN,
+            _ownerSlot,
+            bytes32(uint256(uint160(cfg.governanceTokenOwner())))
+        );
+
+        addrToRemoveFromAlloc.push(address(token));
+    }
+
+    /// @notice This contract has no initializer.
+    function _setL1Block() internal {
+        address impl = _predeployToCodeNamespace(Predeploys.L1_BLOCK_ATTRIBUTES);
+        vm.etch(
+            impl,
+            vm.getDeployedCode("L1Block.sol:L1Block")
+        );
+
+        vm.startPrank(0xDeaDDEaDDeAdDeAdDEAdDEaddeAddEAdDEAd0001);
+        L1Block(Predeploys.L1_BLOCK_ATTRIBUTES).setL1BlockValues(
```

Comment:
We don't need to worry about having these set in genesis. The first block should push the correct values

Reply:
Removed them